src/utils.js

```diff
@@ -81,6 +81,9 @@ var getPrivateIps = function () {
   // Gather candidates for 2 seconds before returning privateIps or timing out
   return new Promise(function (F, R) {
     setTimeout(function () {
+      pc.close().then(function () {
```

Comment:
Cool. I think to be sure to be sure you should call this on failure too, e.g.:
```typescript
var cleanup = () => {
  freedom['core.rtcpeerconnection'].close(pc);
};
pc.close().then(cleanup, cleanup);
```

(my kingdom, etc., for `finally` blocks)